meta/main.yml

```diff
@@ -17,4 +18,11 @@ galaxy_info:
     - kannel
     - smpp
 
-dependencies: []
+dependencies:
+  - role: onaio.monit
+    vars:
+      monit_setup_mode: "{{ kannel_monit_setup_mode }}"
+      monit_scripts: "{{ kannel_monit_scripts }}"
+      monit_opsgenie_api_key: "{{ kannel_monit_opsgenie_api_key }}"
+      slack_monit_endpoint: "{{ kannel_slack_monit_endpoint }}"
```

Comment:
@morrismukiri since the usefulness of the monitoring scripts is pegged on how often Monit should be running them (currently 20 seconds), I think we should also pass `monit_check_interval` with a default value of `20`.